### PR TITLE
Liliths branch

### PR DESCRIPTION
**Pull request by LilithPrimary**
Test pr

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function App(): JSX.Element {
+  return (
+    <>
+      <h1>RSLang</h1>
```
Comment (greg-fullstack) [inline src/App.tsx:6]: What this line code doing?)

**Review by maiklshetinin — APPROVED** [review]

**Review by LilithPrimary — COMMENTED** [review]
Test comment

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
```
Comment (maiklshetinin) [inline src/index.css:2]: ?

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
+{
+  "env": {
```
Comment (maiklshetinin) [inline .eslintrc.json:2]: ??

**Comment by greg-fullstack** [comment]
> Test comment
good job

**Review by greg-fullstack — APPROVED** [review]

**Review by greg-fullstack — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (LilithPrimary) [reply src/App.tsx:6]: О!

**Pull request merged**